site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -217,14 +217,14 @@ The following tables lists the configurable parameters of the Agones chart and t
-| `agones.allocator.pdb.maxUnavailable` | Description of the number of pods from that set that can be unavailable after the eviction. It can be either an absolute number or a percentage.                                           | \`\`    |
-| `agones.ping.pdb.enabled`             | Set to `true` to enable the creation of a [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for the ping deployment                                   | `false` |
-| `agones.ping.pdb.minAvailable`        | Description of the number of pods from that set that must still be available after the eviction, even in the absence of the evicted pod. Can be either an absolute number or a percentage. | `1`     |
-| `agones.ping.pdb.maxUnavailable`      | Description of the number of pods from that set that can be unavailable after the eviction. It can be either an absolute number or a percentage.                                           | \`\`    |
+| Parameter                             | Description                                                                                                                                                                                                                           | Default |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `agones.allocator.pdb.enabled`        | Set to `true` to enable the creation of a [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for the allocator deployment                                                                         | `false` |
+| `agones.allocator.pdb.minAvailable`   | Description of the number of pods from that set that must still be available after the eviction, even in the absence of the evicted pod. Can be either an absolute number or a percentage.                                            | `1`     |
+| `agones.allocator.pdb.maxUnavailable` | Description of the number of pods from that set that can be unavailable after the eviction. It can be either an absolute number or a percentage.                                                                                      | \`\`    |
+| `agones.ping.pdb.enabled`             | Set to `true` to enable the creation of a [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for the ping deployment                                                                              | `false` |
+| `agones.ping.pdb.minAvailable`        | Description of the number of pods from that set that must still be available after the eviction, even in the absence of the evicted pod. Can be either an absolute number or a percentage. **Mutually Exclusive with maxUnavailable** | `1`     |
```

Comment:
Super-nit here and below: Skip the bold, remember the code formatting and fix capitalization, i.e.:

Mutually exclusive with `maxUnavailable`.